project.clj

```diff
@@ -66,7 +66,8 @@
             [jonase/eastwood "0.3.10"]
             [reifyhealth/lein-git-down "0.4.1"]]
   :profiles {:dev     {:dependencies [[clj-http-fake "1.0.3"]]
-                       :resource-paths ["conf/test" "test-resources"]}
+                       :resource-paths ["conf/test" "test-resources"]
+                       :jvm-opts ["-Dotel.javaagent.enabled=false"]}
```

Comment:
I was a little afraid to try to disable the Java agent in the general case even though I was relatively certain that it wouldn't affect runtime behavior. Putting this option in the development profile eliminates most of the errors that we've been seeing, though.